### DISCUSSION TITLE
multiple relay servers

### DISCUSSION THREAD

**Discussion by sgm456** [rustdesk/rustdesk#1959] [General]
Has anyone set up rustdesk with multiple relay servers? how to specify them?
hbbs -r host1,host2
or in some other way?

**Comment by jasoncollege24** [comment]
From my understanding, you either need to have multiple portable versions (each setup on it's own relay server, and can only run one at a time), or you need to manually change the server your client uses. RustDesk doesn't natively support multiple relay servers.

**Comment by danielmarschall** [comment]
I have the same question, but with a different background. If I have IPv4 and IPv6 dual-stack, it would be good if I have "hbbs -r" set to both IPv4 and IPv6, so that the client can choose which fits best...

**Reply by xlionjuan** [reply]
@danielmarschall 
Use a subdomain that has both A and AAAA records.
